common/config/rush/common-versions.json

```diff
@@ -41,12 +41,10 @@
    * This design avoids unnecessary churn in this file.
    */
   "allowedAlternativeVersions": {
-    // "^12.4.1" is required for eventhubs-checkpointstore-blob to use the latest GA version
-    // when there is a new beta version which is being maintained in the repo.
-    // Remove "^12.4.1" when the storage-blob releases a stable version.
-    // Add a new entry in case a new version is being tested through the perf tests (Example: "12.2.0").
-    "@azure/storage-blob": ["^12.4.1"],
     "@azure/ms-rest-js": ["^2.0.0"],
+
+    /** keeping quantum jobs and storage-blob-changefeed out of this hotfix branch hotfix/storage-opentelemetry-ga */
+    "@azure/storage-blob": ["^12.6.0-beta.1", "12.6.0"],
```

Comment:
[minor]: Seems "^12.6.0-beta.1" is not necessary here.

Reply:
In this branch, the quantum jobs package was depending on `^12.6.0-beta.1` and so this was needed to avoid making changes to a package not related to storage